pkgs/tools/networking/p2p/rtorrent-jesec/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, gtest
+, makeWrapper
+, libtorrent-jesec
+, cacert
+, curl
+, ncurses
+, xmlrpc_c
+, nlohmann_json
+, xmlRpcSupport ? true
+, jsonRpcSupport ? true
```

Comment:
Do we even need those options?

Reply:
They're useful options to have. For example, in my configuration, I purely want JSON-RPC, so I'll disable the XML-based one via an override.

Reply:
We usually do not include such options because it increases complexity and the options are not often tested.

Reply:
I think it's fine here since their implementation is very simple.